### PR TITLE
chore(stream-caip10-link-handler): Include addresses in error message on mismatch between controller and address who signed the update

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Comment by stbrody** [comment]
build is broken currently so I can't make sure this didn't break any tests until it is fixed

**Review by stephhuynh18 — APPROVED** [review]
LGTM with a small question/suggestion

File: packages/canary-integration/src/__tests__/caip10/caip-flows.ts [hunk 1]
```diff
@@ -10,12 +11,23 @@ export async function happyPath(ceramic: CeramicApi, authProvider: AuthProvider)
...
-  accountId.address = 'wrong-test-user'
-  const caip = await Caip10Link.fromAccount(ceramic, accountId)
+export async function wrongProof(
+  ceramic: CeramicApi,
+  authProvider: AuthProvider,
+  wrongAccountId?: AccountId
+) {
+  const signingAccountId = await authProvider.accountId()
+  if (!wrongAccountId) {
+    wrongAccountId = await authProvider.accountId()
```
[the remark this answers is not in the document]
Comment (stbrody) [reply packages/canary-integration/src/__tests__/caip10/caip-flows.ts:21]: They result in the same account ID, but different *instances* of the `AccountID` object.  For wrongAccountId we modify the object after loading it to make it wrong.  So we need two independent Objects that don't share memory.  The other option would be trying to do a deepClone, but doing that without losing the type data and converting it into a raw Object would be tricky, so just loading two instances by calling the `accountId()` method twice was the easiest way.  IDK if @ukstv knows of any other cleaner way to handle this?

[the remark this answers is not in the document]
Comment (ukstv) [reply packages/canary-integration/src/__tests__/caip10/caip-flows.ts:21]: I do not know of a better way. Seems legit to me.

**Review by ukstv — APPROVED** [review]

**Pull request merged**